tests/cli/output/show_config.txt

```diff
@@ -1,2224 +1,2222 @@
 
+
```

Comment:
strange

Reply:
Maybe... `mini_[n]d_ucwd.txt` are the same, though.

It looks like this happens because old `--show-config` started with:
```
# Uncrustify-<version>
#
# General options
#

```
...while `--update-config-with-doc` starts with:
```
# Uncrustify-<version>

#
# General options
#

```
(Note the extra blank line after the uncrustify version.)